todo/src/components/Form.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {addTask} from '../actions';
+import './todo.css'
+
+class Form extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            todo: ''  
+    }}
+
+    handleInputChange = e => {
+        this.setState({todo: e.target.value})
+      };
+
+      addTodo = e => {
+          e.preventDefault();
+          this.props.addTask(this.state.todo)
+          this.setState({todo: ''})
+          document.querySelector('#input').value = ''
```

Comment:
You shouldn't need to use the `querySelector` statement here - just change your input field to use the value of something that is already defined in state (for this case, use `this.state.todo`).